orm/model/ormtable/auto_increment_test.go

```diff
@@ -25,24 +26,26 @@ func TestAutoIncrementScenario(t *testing.T) {
 	assert.Assert(t, ok)
 
 	// first run tests with a split index-commitment store
-	runAutoIncrementScenario(t, autoTable, ormtable.WrapContextDefault(testkv.NewSplitMemBackend()))
+	runAutoIncrementScenario(ormtable.WrapContextDefault(testkv.NewSplitMemBackend()), t, autoTable)
 
 	// now run with shared store and debugging
 	debugBuf := &strings.Builder{}
 	store := testkv.NewDebugBackend(
 		testkv.NewSharedMemBackend(),
 		&testkv.EntryCodecDebugger{
 			EntryCodec: table,
-			Print:      func(s string) { debugBuf.WriteString(s + "\n") },
+			Print:      func(s string) { debugBuf.WriteString(s + "\n") }, //nolint:errcheck // ignore errors
 		},
 	)
-	runAutoIncrementScenario(t, autoTable, ormtable.WrapContextDefault(store))
+	runAutoIncrementScenario(ormtable.WrapContextDefault(store), t, autoTable)
 
 	golden.Assert(t, debugBuf.String(), "test_auto_inc.golden")
 	checkEncodeDecodeEntries(t, table, store.IndexStoreReader())
 }
 
-func runAutoIncrementScenario(t *testing.T, table ormtable.AutoIncrementTable, ctx context.Context) {
+// runAutoIncrementScenario runs a simple scenario with an auto-increment table.
+func runAutoIncrementScenario(ctx context.Context, t *testing.T, table ormtable.AutoIncrementTable) {
```

Comment:
I am not for swapping argument of functions in style PRs

Reply:
this complies with a linter rule, actually two of them:

* revive: context-as-argument
* thelper